articles/django-app-using-postgresql-database/index.md

```diff
@@ -0,0 +1,248 @@
+Django is a high-level Python Web framework that encourages rapid development and clean, pragmatic design. Built by experienced developers, it takes care of much of the hassle of Web development, so you can focus on writing your app without needing to reinvent the wheel. It’s free and open-source <cite>[Django](https://www.djangoproject.com)<cite>
+
+PostgreSQL is a powerful, open-source object-relational database system that uses and extends the SQL language combined with many features that safely store and scale the most complicated data workloads. <cite>[PostgreSQL](https://www.postgresql.org/about/)<cite>
+
+### Prerequisites
+To follow this tutorial you need to have:
+
+ - [Python3](https://www.python.org/downloads/)
+ 
+ - [PostgreSQL](https://www.postgresql.org/download/windows/)
+ 
+ - [virtualenv](https://pypi.org/project/virtualenv/)
+
+### Step 1: Creating our database through the command line
+Open the PostgreSQL shell. You can find the PSQL Shell in the Start Menu.
+
+![access psql](/engineering-education/django-app-using-postgresql-database/access_psql.png)
+
+![psql_shell](/engineering-education/django-app-using-postgresql-database/psql_shell.png)
+
+Leave the **Server, Database, Port, Username** options empty. By clicking on `Enter` in the keyboard, you have accepted the default value provided by Shell. Provide the **Password** input with the Password you used during the PostgreSQL Installation on your machine. You will see a similar result in your PSQL shell like the one the above image if correctly implemented.
+
+```bash 
+ CREATE DATABASE helloworld;
+ CREATE USER <yourname> WITH PASSWORD '<password>';
+```
```

Comment:
Briefly explain what these commands are.

Reply:
It would be great if you split this into two code snippets and explain them seperately.